src/Modals/Modal.tsx

```diff
@@ -75,6 +75,10 @@ export interface ModalProps {
    * The content to be displayed within the modal
    */
   children: ReactNode;
+  /**
+   * Identifies the element that provides the modal
+   */
+  ariaLabelledBy: string;
```

Comment:
I think this is missing a test case

Reply:
I added a test to check the value of the aria-labelledBy property through commit 0a12491.